src/main/kotlin/domain/MoveStrategy.kt

```diff
@@ -0,0 +1,16 @@
+package domain
+
+import java.util.*
+
+private val random = Random()
+private const val CRITERIA = 4
+
+enum class MoveStrategy(val description: String, private val function: () -> Int) {
+    RANDOM("랜덤하게 가거나, 혹은 가지 않는 전략", { random.nextInt() }),
+    STAND_STILL("항상 움직이지 않는 전략", { 4 }),
+    ALWAYS_GO("항상 움직이는 전략", { 5 });
+
+    fun isMove(): Boolean {
+        return function.invoke() > CRITERIA
```

Comment:
```suggestion
        return function() > CRITERIA
```
`invoke`는 `()`을 나타내는 operator function입니다